### PR TITLE
[CFM-1380] -Bootstrap's tooltip JS in activity stream messages.

### PR DESCRIPTION
**Pull request by ybaras**
https://issuetracker.amplexor.com/jira/browse/CFM-1380

### CODE REVIEW

**Comment by ybaras** [comment]
tt

File: project/profiles/capacity4more/modules/c4m/restful/c4m_restful_quick_post/components/c4m-app/src/controllers/activity.js [hunk 1]
```diff
@@ -8,6 +8,15 @@
 angular.module('c4mApp')
   .controller('ActivityCtrl', function($scope, DrupalSettings, EntityResource, $timeout, $interval, $sce) {
 
+    /*
+     * Init the Bootstrap tooltips.
+     */
+    $scope.bindBoostrapTooltips = function() {
+      $timeout(function() {
```
Comment (ordavidil) [inline project/profiles/capacity4more/modules/c4m/restful/c4m_restful_quick_post/components/c4m-app/src/controllers/activity.js]: @ybaras 
Please add a comment about why we're using the `$timeout`.
thanks.

**Comment by soniCaH** [comment]
Did a clean install and function is not executed.
I see functionality in {{project/profiles/capacity4more/modules/c4m/restful/c4m_restful_quick_post/components/c4m-app/src/controllers/activity.js}} and {{project/profiles/capacity4more/modules/c4m/restful/c4m_restful_quick_post/components/c4m-app/dist/c4m-app.js}} but NOT in {{project/profiles/capacity4more/libraries/bower_components/c4m-app/dist/c4m-app.js}}
Maybe related to CFM-1390?

**Comment by jover** [comment]
The functionality written in this pull request actually works, but as far as I can see the output of the generated/concatenated JS is not written to the file which Drupal uses.

So the changes in the file _project/profiles/capacity4more/modules/c4m/restful/c4m_restful_quick_post/components/c4m-app/src/controllers/activity.js_ are to be found in the file _project/profiles/capacity4more/modules/c4m/restful/c4m_restful_quick_post/components/c4m-app/dist/c4m-app.js_ but Drupal uses _project/profiles/capacity4more/libraries/bower_components/c4m-app/dist/c4m-app.js_

**Comment by ordavidil** [comment]
@jover 
It seems related to https://issuetracker.amplexor.com/jira/browse/CFM-1480
and it stall some other issues as well.
This is something related to the installation of the project so should be taking care of over there, we spent a lot of hours trying to fix that but since we didn't write the installation then it is more complex for us to handle this one, at this point I would copy the file manually (since this PR is only adding some lines to an existing file).

**Pull request merged**